$NetBSD$

Remove "test" from directory list.
Add rule to build tests on request.

--- Makefile.orig	2024-03-18 22:51:05.000000000 +0000
+++ Makefile
@@ -16,7 +16,7 @@
 
 include makeinclude
 
-DIRS = $(IMAGEDIRS) $(LIBDECORDIR) src $(CAIRODIR) $(FLUIDDIR) fltk-options test \
+DIRS = $(IMAGEDIRS) $(LIBDECORDIR) src $(CAIRODIR) $(FLUIDDIR) fltk-options \
        documentation
 
 all: makeinclude fltk-config
@@ -25,6 +25,11 @@ all: makeinclude fltk-config
 		(cd $$dir; $(MAKE) $(MFLAGS)) || exit 1;\
 	done
 
+# [pkgsrc] Build tests only on request
+check: makeinclude
+	echo "=== making test ===";\
+	(cd test; $(MAKE) $(MFLAGS)) || exit 1;\
+
 install: makeinclude
 	-mkdir -p $(DESTDIR)$(bindir)
 	$(RM) $(DESTDIR)$(bindir)/fltk-config
